sources/packages/backend/libs/test-utils/src/factories/disbursement-schedule.ts

```diff
@@ -34,3 +41,24 @@ export function createFakeDisbursementSchedule(relations?: {
   schedule.disbursementScheduleStatus = DisbursementScheduleStatus.Pending;
   return schedule;
 }
+
+/**
+ * Creates disbursement schedule awards based on the given schedule and disbursement number.
+ * @param disbursementSchedule Disbursement schedule containing disbursement values.
+ * @param disbursementNumber Number associated with the disbursement schedule (e.g., 1 for the first, 2 for the second).
+ * @returns Disbursement awards with keys like 'disbursement1code' and values as amounts.
+ */
+export function createDisbursementScheduleAwards(
+  disbursementSchedule: DisbursementSchedule,
+  disbursementNumber: number,
+): DisbursementAwards {
+  const disbursementScheduleAwards: DisbursementAwards = {};
+
+  disbursementSchedule.disbursementValues.forEach((disbursementValue) => {
+    disbursementScheduleAwards[
+      `disbursement${disbursementNumber}${disbursementValue.valueCode.toLowerCase()}`
```

Comment:
Is there a possibility of having a repeated `valueCode` creating a key conflict (duplicate key)?

Reply:
No there will not be any